diff -u -Nru5 hdf5-1.10.3/config/cmake_ext_mod/ConfigureChecks.cmake hdf5-1.10.3.msvc-non-vs-generator/config/cmake_ext_mod/ConfigureChecks.cmake
--- hdf5-1.10.3/config/cmake_ext_mod/ConfigureChecks.cmake	2018-08-09 17:36:31.000000000 -0400
+++ hdf5-1.10.3.msvc-non-vs-generator/config/cmake_ext_mod/ConfigureChecks.cmake	2019-03-14 10:08:32.051965687 -0400
@@ -62,15 +62,10 @@
 # WINDOWS Hard code Values
 # ----------------------------------------------------------------------
 
 set (WINDOWS)
 if (WIN32)
-  if (MINGW)
-    set (${HDF_PREFIX}_HAVE_MINGW 1)
-    set (WINDOWS 1) # MinGW tries to imitate Windows
-    set (CMAKE_REQUIRED_FLAGS "-DWIN32_LEAN_AND_MEAN=1 -DNOGDI=1")
-  endif ()
   set (${HDF_PREFIX}_HAVE_WIN32_API 1)
   set (CMAKE_REQUIRED_LIBRARIES "ws2_32.lib;wsock32.lib")
   if (NOT UNIX AND NOT MINGW)
     set (WINDOWS 1)
     set (CMAKE_REQUIRED_FLAGS "/DWIN32_LEAN_AND_MEAN=1 /DNOGDI=1")
@@ -86,13 +81,11 @@
   set (${HDF_PREFIX}_HAVE_SYS_TYPES_H 1)
   set (${HDF_PREFIX}_HAVE_LIBM 1)
   set (${HDF_PREFIX}_HAVE_STRDUP 1)
   set (${HDF_PREFIX}_HAVE_SYSTEM 1)
   set (${HDF_PREFIX}_HAVE_LONGJMP 1)
-  if (NOT MINGW)
-    set (${HDF_PREFIX}_HAVE_GETHOSTNAME 1)
-  endif ()
+  set (${HDF_PREFIX}_HAVE_GETHOSTNAME 1)
   if (NOT UNIX AND NOT CYGWIN AND NOT MINGW)
     set (${HDF_PREFIX}_HAVE_GETCONSOLESCREENBUFFERINFO 1)
   endif ()
   set (${HDF_PREFIX}_HAVE_FUNCTION 1)
   set (${HDF_PREFIX}_GETTIMEOFDAY_GIVES_TZ 1)
